<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpParameterWindow</class>
 <widget class="QDialog" name="TcpParameterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonDeconnexion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonConnexion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>24</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>54</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.128.3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>7538</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="color_box_conn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_conn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>144</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion state</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
